### ISSUE TITLE
`dotnet new --install` of SpaTemplates fails with `no matches found` error on Zsh (macOS)

### ISSUE THREAD

**Issue by Meligy** [dotnet/cli#6014]
## Steps to reproduce

1. Install dotnet SDK on Mac (version details in `dotnet --info` at the end)
2. Start a new Zsh terminal
3. Run the following command in Zsh:
  
  ```
dotnet new --install Microsoft.AspNetCore.SpaTemplates::*
```

## Expected  behavior

The CLI downloads the templates, and reports installing them, then lists them in the available templates for `dotnet new`

## Actual behavior

The following message is shown almost instantly:

```
zsh: no matches found: Microsoft.AspNetCore.SpaTemplates::*
```

## Environment data

`dotnet --info` output:

```
> dotnet --info   
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.1
```

### Additional Information

This works fine if you start `bash` inside Zsh and run the command inside bash.

**Comment by sayedihashimi** [comment]
Do you have OpenSSL installed?

**Comment by Meligy** [comment]
Yes, and ensured it is properly linked.

I'd expect more severe-looking errors if it's an OpenSSL failure too.

**Comment by Meligy** [comment]
However, any specific test you'd like me to do to time out wrong version linked etc?

Thanks heaps.

**Comment by spboyer** [comment]
@meligy - noticed that you are using zsh, have you tries using/testing in just terminal to isolate if this is just a zsh bug?

**Comment by Meligy** [comment]
Yes, just zsh. I ran `bash` inside Zsh and repeated the command and it worked just fine. I was able to find the templates listed after restarting the Zsh terminal and confirming with `dotnet new --help`.

I updated the issue title and description to reflect that.

**Comment by dasMulli** [comment]
@Meligy does `dotnet new --install "Microsoft.AspNetCore.SpaTemplates::*"` work?

**Comment by Meligy** [comment]
It seems to work yes.

I get the following error, but I also get it in bash, so, I expect it's because I have already installed the templates (not sure what's the flag to remove them).

At least both are giving the same output, and it seems to be doing something.

```
> dotnet new --install "Microsoft.AspNetCore.SpaTemplates::*"
  Restoring packages for /Users/Meligy/.templateengine/dotnetcli/v1.0.1/scratch/restore.csproj...
  Installing System.Xml.XPath 4.0.1.
  Installing runtime.native.System.IO.Compression 4.1.0.
  Installing Microsoft.NETCore.DotNetHost 1.0.1.
  Installing Microsoft.NETCore.Targets 1.0.1.
  Installing System.Xml.XPath.XDocument 4.0.1.
  Installing System.Xml.XmlDocument 4.0.1.
  Installing System.Text.Encoding.CodePages 4.0.1.
  Installing System.Diagnostics.StackTrace 4.0.1.
  Installing System.Diagnostics.FileVersionInfo 4.0.0.
  Installing Microsoft.CodeAnalysis.Analyzers 1.1.0.
  Installing System.Security.Cryptography.Csp 4.0.0.
  Installing System.Security.Cryptography.Cng 4.2.0.
  Installing System.Security.Principal 4.0.1.
  Installing Microsoft.NETCore.App 1.0.4.
  Installing Microsoft.AspNetCore.SpaTemplates 0.8.6.
  Installing System.Security.Claims 4.0.1.
  Installing Libuv 1.9.1.
  Installing runtime.native.System.Net.Security 4.0.1.
  Installing System.Security.Principal.Windows 4.0.0.
  Installing Microsoft.CodeAnalysis.CSharp 1.3.0.
  Installing Microsoft.CSharp 4.0.1.
  Installing Microsoft.CodeAnalysis.VisualBasic 1.3.0.
  Installing Microsoft.NETCore.DotNetHostPolicy 1.0.3.
  Installing Microsoft.NETCore.Platforms 1.0.2.
  Installing Microsoft.NETCore.Runtime.CoreCLR 1.0.6.
  Installing Microsoft.VisualBasic 10.0.1.
  Installing NETStandard.Library 1.6.0.
  Installing System.Buffers 4.0.0.
  Installing System.Collections.Immutable 1.2.0.
  Installing System.ComponentModel 4.0.1.
  Installing System.ComponentModel.Annotations 4.1.0.
  Installing System.Diagnostics.DiagnosticSource 4.0.0.
  Installing System.Diagnostics.Process 4.1.0.
  Installing System.Dynamic.Runtime 4.0.11.
  Installing System.Globalization.Extensions 4.0.1.
  Installing System.IO.FileSystem.Watcher 4.0.0.
  Installing System.IO.MemoryMappedFiles 4.0.0.
  Installing System.IO.UnmanagedMemoryStream 4.0.1.
  Installing System.Linq.Expressions 4.1.1.
  Installing System.Linq.Parallel 4.0.1.
  Installing System.Linq.Queryable 4.0.1.
  Installing System.Net.NameResolution 4.0.0.
  Installing System.Net.Requests 4.0.11.
  Installing System.Net.Http 4.1.1.
  Installing System.Net.Security 4.0.0.
  Installing System.Net.WebHeaderCollection 4.0.1.
  Installing System.Numerics.Vectors 4.1.1.
  Installing System.Security.Cryptography.OpenSsl 4.0.0.
  Installing runtime.native.System.Security.Cryptography 4.0.0.
  Installing System.Reflection.DispatchProxy 4.0.1.
  Installing runtime.native.System.Net.Http 4.0.1.
  Installing System.Reflection.Metadata 1.3.0.
  Installing System.Reflection.Emit.Lightweight 4.0.1.
  Installing System.Reflection.TypeExtensions 4.1.0.
  Installing System.Resources.Reader 4.0.0.
  Installing System.Threading.Overlapped 4.0.1.
  Installing System.Runtime.Loader 4.0.0.
  Installing System.Security.Cryptography.Algorithms 4.2.0.
  Installing System.Security.Cryptography.Encoding 4.0.0.
  Installing System.Security.Cryptography.Primitives 4.0.0.
  Installing System.Reflection.Emit.ILGeneration 4.0.1.
  Installing System.Security.Cryptography.X509Certificates 4.1.0.
  Installing System.Threading.Tasks.Dataflow 4.6.0.
  Installing System.Threading.Tasks.Extensions 4.0.0.
  Installing System.Threading.Tasks.Parallel 4.0.1.
  Installing System.Threading.Thread 4.0.0.
  Installing System.Threading.ThreadPool 4.0.10.
  Installing Microsoft.CodeAnalysis.Common 1.3.0.
  Installing System.Collections 4.0.11.
  Installing System.Diagnostics.Debug 4.0.11.
  Installing System.Globalization 4.0.11.
  Installing System.Linq 4.1.0.
  Installing System.ObjectModel 4.0.12.
  Installing System.Reflection 4.1.0.
  Installing System.Reflection.Extensions 4.0.1.
  Installing System.Reflection.Primitives 4.0.1.
  Installing System.Resources.ResourceManager 4.0.1.
  Installing System.Runtime 4.1.0.
  Installing System.Runtime.Extensions 4.1.0.
  Installing System.Runtime.InteropServices 4.1.0.
  Installing System.Threading 4.0.11.
  Installing Microsoft.NETCore.DotNetHostResolver 1.0.1.
  Installing Microsoft.NETCore.Jit 1.0.6.
  Installing Microsoft.NETCore.Windows.ApiSets 1.0.1.
  Installing Microsoft.Win32.Primitives 4.0.1.
  Installing System.AppContext 4.1.0.
  Installing System.Collections.Concurrent 4.0.12.
  Installing System.Console 4.0.0.
  Installing System.Diagnostics.Tools 4.0.1.
  Installing System.Diagnostics.Tracing 4.1.0.
  Installing System.Reflection.Emit 4.0.1.
  Installing System.Globalization.Calendars 4.0.1.
  Installing System.IO 4.1.0.
  Installing System.IO.Compression 4.1.0.
  Installing System.IO.Compression.ZipFile 4.0.1.
  Installing System.IO.FileSystem 4.0.1.
  Installing System.IO.FileSystem.Primitives 4.0.1.
  Installing System.Net.Primitives 4.0.11.
  Installing System.Net.Sockets 4.1.0.
  Installing System.Runtime.Handles 4.0.1.
  Installing System.Runtime.InteropServices.RuntimeInformation 4.0.0.
  Installing System.Runtime.Numerics 4.0.1.
  Installing System.Text.Encoding 4.0.11.
  Installing System.Text.Encoding.Extensions 4.0.11.
  Installing System.Text.RegularExpressions 4.1.0.
  Installing System.Threading.Tasks 4.0.11.
  Installing System.Threading.Timer 4.0.1.
  Installing System.Xml.ReaderWriter 4.0.11.
  Installing System.Xml.XDocument 4.0.11.
  Installing Microsoft.Win32.Registry 4.0.0.
  Installing runtime.native.System 4.0.0.
/usr/local/share/dotnet/sdk/1.0.1/NuGet.targets(97,5): error : Too many open files [/Users/Meligy/.templateengine/dotnetcli/v1.0.1/scratch/restore.csproj]
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.xml.xdocument.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.xml.readerwriter.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.timer.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.threadpool.4.0.10.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.thread.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.tasks.parallel.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.tasks.extensions.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.tasks.dataflow.4.6.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.tasks.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.overlapped.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.threading.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.text.regularexpressions.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.text.encoding.extensions.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.text.encoding.codepages.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.text.encoding.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.principal.windows.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.principal.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.x509certificates.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.primitives.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.openssl.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.encoding.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.csp.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.cng.4.2.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.security.cryptography.algorithms.4.2.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.numerics.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.loader.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.interopservices.runtimeinformation.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.interopservices.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.handles.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.extensions.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.runtime.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.resources.resourcemanager.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.resources.reader.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.typeextensions.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.primitives.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.metadata.1.3.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.extensions.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.emit.lightweight.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.emit.ilgeneration.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.emit.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.dispatchproxy.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.reflection.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.objectmodel.4.0.12.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.numerics.vectors.4.1.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.webheadercollection.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.sockets.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.security.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.requests.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.primitives.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.nameresolution.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.net.http.4.1.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.linq.queryable.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.linq.parallel.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.linq.expressions.4.1.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.linq.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.unmanagedmemorystream.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.memorymappedfiles.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.filesystem.watcher.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.filesystem.primitives.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.filesystem.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.compression.zipfile.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.compression.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.io.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.globalization.extensions.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.globalization.calendars.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.globalization.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.dynamic.runtime.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.tracing.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.tools.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.process.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.fileversioninfo.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.diagnosticsource.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.diagnostics.debug.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.console.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.componentmodel.annotations.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.componentmodel.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.collections.immutable.1.2.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.collections.concurrent.4.0.12.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.collections.4.0.11.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.buffers.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/system.appcontext.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/runtime.native.system.security.cryptography.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/runtime.native.system.net.security.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/runtime.native.system.net.http.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/runtime.native.system.io.compression.4.1.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/runtime.native.system.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/netstandard.library.1.6.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.win32.registry.4.0.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.win32.primitives.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.visualbasic.10.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.windows.apisets.1.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.targets.1.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.runtime.coreclr.1.0.6.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.platforms.1.0.2.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.jit.1.0.6.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.dotnethostresolver.1.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.dotnethostpolicy.1.0.3.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.netcore.dotnethost.1.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.csharp.4.0.1.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.codeanalysis.visualbasic.1.3.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.codeanalysis.csharp.1.3.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/microsoft.codeanalysis.common.1.3.0.nupkg" is not a valid package specification
Error: Package "/Users/Meligy/.templateengine/dotnetcli/v1.0.1/packages/libuv.1.9.1.nupkg" is not a valid package specification
Too many open files
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Interop.CheckIo[TSafeHandle](TSafeHandle handle, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
   at System.IO.UnixFileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)
   at System.IO.UnixFileSystem.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.File.Open(String path, FileMode mode, FileAccess access, FileShare share)
   at Microsoft.TemplateEngine.Utils.PhysicalFileSystem.ReadAllText(String path)
   at Microsoft.TemplateEngine.Edge.Paths.ReadAllText(String path, String defaultValue)
   at Microsoft.TemplateEngine.Edge.Settings.TemplateCache.LoadTemplateCacheForLocale(String locale)
   at Microsoft.TemplateEngine.Edge.Settings.TemplateCache.WriteTemplateCacheForLocale(String locale)
   at Microsoft.TemplateEngine.Edge.Settings.TemplateCache.WriteTemplateCaches()
   at Microsoft.TemplateEngine.Cli.Installer.InstallLocalPackages(IReadOnlyList`1 packageNames)
   at Microsoft.TemplateEngine.Cli.Installer.InstallRemotePackages(List`1 packages)
   at Microsoft.TemplateEngine.Cli.Installer.InstallPackages(IEnumerable`1 installationRequests)
   at Microsoft.TemplateEngine.Cli.New3Command.EnterInstallFlowAsync()
   at Microsoft.TemplateEngine.Cli.New3Command.<EnterMaintenanceFlowAsync>d__63.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Microsoft.TemplateEngine.Cli.New3Command.<ExecuteAsync>d__66.MoveNext()
```

**Comment by dasMulli** [comment]
`error : Too many open files` => https://github.com/NuGet/Home/issues/2163
Looks like this didn't happen often because the CLI ships package cache archives but the templates performs a fresh restore.
(Workaround is `ulimit -n 512`)

**Comment by livarcocc** [comment]
Given that we have a workaround and that the actual restore issue is already tracked by NuGet, I am closing this one.

**Comment by blackie1019** [comment]
![screen shot 2017-03-26 at 10 38 41 pm](https://cloud.githubusercontent.com/assets/824412/24332212/4bfd3de0-1275-11e7-99d4-1beb1b2ba5aa.png)

I have same issue but different exception, can anyone help me to solve it?

**Comment by sayedihashimi** [comment]
Try

    dotnet new --install "Microsoft.AspNetCore.SpaTemplates::*"

**Comment by rrelyea** [comment]
From @mlorbetske:
that issue was fixed as of the 1.0.4 CLI. Either updating to a later CLI build or raising the value of ulimit should fix it.
